sale_stock_available_to_promise_release/models/sale_order_line.py

```diff
@@ -90,6 +80,9 @@ def _get_availability_data(self):
         elif float_compare(available_qty, 0, precision_rounding=rounding) == 1:
             availability_status = "partial"
             delayed_qty = self.product_uom_qty - available_qty
+        # On order product
+        elif self.is_mto:
+            availability_status = "on_order"
```

Comment:
shouldn't this be checked first ?
We might have MTO sale order lines availability status set to partial here.

Reply:
It depends what we want to show to the user first. I would show 1st partial (we are able to deliver some qty), 2nd on_order ( we have no quantity available and need to fully order it)

Reply:
If mto, it's on order, period, whether you have stock or not.

Reply:
If user wants smth else, he changes the route on the so line, and that's it.

Reply:
Sale person wants to know if product is available to ship or if action is required to order it. Otherwise this status is useless for him.

Reply:
If you want to display availability, then product shouldn't be MTO.
@jbaudoux @santostelmo can we discuss this ? I'm not sure to understand what we're solving here...